sklearn/metrics/ranking.py

```diff
@@ -47,13 +47,18 @@ def auc(x, y, reorder=False):
     Parameters
     ----------
     x : array, shape = [n]
-        x coordinates.
+        Increasing or decreasing x coordinates.
```

Comment:
How about "x coordinates. These must be either monotonic increasing or monotonic decreasing."? Otherwise, this seems vague.